### PR TITLE
Package coq-itauto.8.13

### PR DESCRIPTION
**Pull request by fajb**

### CODE REVIEW

File: released/packages/coq-itauto.8.13/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+bug-reports: "frederic.besson@inria.fr"
+license: "GPL 3"
+synopsis: "'itauto' is a reflexive SAT solver parameterised by a leaf tactic"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Cdcl"]
```
Comment (clarus) [inline released/packages/coq-itauto.8.13/opam:15]: Not needed with opam 2 (probably the first line should specify `opam-version: "2.0"` too unless compatibility with opam 1 is needed). Most packages are requiring opam 2 now.

File: released/packages/coq-itauto.8.13/opam [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+depends: [
+  "ocaml" {>= "4.9~"}
+  "dune" {>= "2.7.1"}
+  "coq" {>= "8.13.~" }
+  "ocamlbuild" {build }
```
Comment (clarus) [inline released/packages/coq-itauto.8.13/opam:20]: Wondering if `dune` and `ocamlbuild` are needed as dependencies, as I do not see them in the Makefile (but I probably miss something).

File: released/packages/coq-itauto.8.13/opam [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+depopts: [ "ocamlformat" {build} ]
+
+url {
+  src: "https://gitlab.inria.fr/fbesson/itauto/-/archive/8.13/itauto-8.13.tar.gz"
+  checksum: "sha256=971ac9da9ff16ba83ad0236a5206ce0b8617e62c026340441fea0f4f80b79ded"
+}
+  
+tags: [
+  "keyword:integers" "keyword:SAT" "keyword:SMT" "keyword:automation"
+  "logpath:Cdcl"
```
Comment (clarus) [inline released/packages/coq-itauto.8.13/opam:31]: :ok_hand:

File: released/packages/coq-itauto.8.13/opam (hunk 1 above)
Comment (fajb) [reply released/packages/coq-itauto.8.13/opam:15]: Yes. I ran opam lint but forgot to commit.

File: released/packages/coq-itauto.8.13/opam (hunk 2 above)
Comment (fajb) [reply released/packages/coq-itauto.8.13/opam:20]: There is a sub-makefile using `ocamlbuild`.
For `dune`, I don't remember. There used to be a dependency problem with coq. I'll check.
Thanks.

**Comment by clarus** [comment]
Top!

**Comment by fajb** [comment]
Thanks a alot.

**Pull request merged**